README.md

```diff
@@ -2,6 +2,16 @@
 An operator to support OKD aggregated cluster logging.  Cluster logging configuration information
 is found in the [configuration](./docs/configuration.md) documentation.
 
+## Quick Start
+To get started with cluster logging and the `cluster-logging-operator`:
+* Ensure Docker is installed on your local system
+```
+$ oc login $CLUSTER -u $ADMIN_USER -p $ADMIN_PASSWD
+$ REMOTE_CLUSTER=true make deploy-example
```

Comment:
do you need `REMOTE_CLUSTER` ? Or does this assume that $CLUSTER is a remote cluster

Reply:
You need `REMOTE_CLUSTER`